### ISSUE TITLE
Some general issues with assignment 1

### ISSUE THREAD

**Issue by matthew-w-lundy** [JIAO-Hao-mcgill/JIAO-Hao-HW#1]
You have to answer the questions in the problems and not just the coding. For the next assignment try to attach a PDF with short answers to all of the questions in the problems in addition to your code. This should also allow you to post your code output in the same document. Your code is good but you're losing free marks because of this.

**Comment by matthew-w-lundy** [comment]
I know you used a pdf for q1, but for q3 for example you are losing ~50% because of missing info because you didn't write answers to the questions

**Comment by JIAO-Hao-mcgill** [comment]
Thank you for your correction!
I will write a pdf for the homework next time. In fact, I write some comments in the code like the estimate of some errors. About q3, I am sorry that I have though I just need show the differences by figures. Can I add my answer now to make up?
By the way, it is the first time for me to write code by python, I do not know how to how to post my code output in the same document. Does that means I print pictures and put them together by latex?

Thank you very much!
JIAO Hao
________________________________
发件人: Matthew Lundy <notifications@github.com>
发送时间: 2020年9月21日 23:14
收件人: JIAO-Hao-mcgill/JIAO-Hao-HW <JIAO-Hao-HW@noreply.github.com>
抄送: Subscribed <subscribed@noreply.github.com>
主题: Re: [JIAO-Hao-mcgill/JIAO-Hao-HW] Some general issues with assignment 1 (#1)


I know you used a pdf for q1, but for q3 for example you are losing ~50% because of missing info because you didn't write answers to the questions

―

**Comment by matthew-w-lundy** [comment]
Yes, so it is primarily issued with the problems that require justification. For example in q3:"What should the error be for the Lorentzian from the rational function fit? Does what you got agree with your expectations when the order is higher?", since you provided no answer to this estimate I can't give you any points. This was meant to be a apriori estimate.    I'm sorry it's a little past the deadline now for you to get full credit but if you want to upload a written solution I can give you partial credit on those questions.  

Additionally "Can  you  understand  what  has happened by looking at p and q?" No answer or code was provided.

**Comment by JIAO-Hao-mcgill** [comment]
Really thank you! I think partial credit is fine! I will upload the answers of these questions today (I will try my best to finish it before today's class and I will inform you once I upload)!
    About the last question, I think I misunderstand it before. I have thought it is ask about the result of different n,m (I confused pq & nm before. Sorry for that!)
________________________________
发件人: Matthew Lundy <notifications@github.com>
发送时间: 2020年9月21日 23:49
收件人: JIAO-Hao-mcgill/JIAO-Hao-HW <JIAO-Hao-HW@noreply.github.com>
抄送: Hao Jiao <hao.jiao@mail.mcgill.ca>; Comment <comment@noreply.github.com>
主题: Re: [JIAO-Hao-mcgill/JIAO-Hao-HW] Some general issues with assignment 1 (#1)


Yes, so it is primarily issued with the problems that require justification. For example in q3:"What should the error be for the Lorentzian from the rational function fit? Does what you got agree with your expectations when the order is higher?", since you provided no answer to this estimate I can't give you any points. This was meant to be a apriori estimate. I'm sorry it's a little past the deadline now for you to get full credit but if you want to upload a written solution I can give you partial credit on those questions.

Additionally "Can you understand what has happened by looking at p and q?" No answer or code was provided.

―

**Comment by JIAO-Hao-mcgill** [comment]
By the way, I just need rewrite problem3 or all problems?
________________________________
发件人: Hao Jiao <hao.jiao@mail.mcgill.ca>
发送时间: 2020年9月21日 23:57
收件人: JIAO-Hao-mcgill/JIAO-Hao-HW <reply@reply.github.com>
主题: 回复: [JIAO-Hao-mcgill/JIAO-Hao-HW] Some general issues with assignment 1 (#1)

    Really thank you! I think partial credit is fine! I will upload the answers of these questions today (I will try my best to finish it before today's class and I will inform you once I upload)!
    About the last question, I think I misunderstand it before. I have thought it is ask about the result of different n,m (I confused pq & nm before. Sorry for that!)
________________________________
发件人: Matthew Lundy <notifications@github.com>
发送时间: 2020年9月21日 23:49
收件人: JIAO-Hao-mcgill/JIAO-Hao-HW <JIAO-Hao-HW@noreply.github.com>
抄送: Hao Jiao <hao.jiao@mail.mcgill.ca>; Comment <comment@noreply.github.com>
主题: Re: [JIAO-Hao-mcgill/JIAO-Hao-HW] Some general issues with assignment 1 (#1)


Yes, so it is primarily issued with the problems that require justification. For example in q3:"What should the error be for the Lorentzian from the rational function fit? Does what you got agree with your expectations when the order is higher?", since you provided no answer to this estimate I can't give you any points. This was meant to be a apriori estimate. I'm sorry it's a little past the deadline now for you to get full credit but if you want to upload a written solution I can give you partial credit on those questions.

Additionally "Can you understand what has happened by looking at p and q?" No answer or code was provided.

―

**Comment by matthew-w-lundy** [comment]
I have only been going through the grades on q3 at the moment but all questions should have written answers or points will be lost in most cases.

**Comment by JIAO-Hao-mcgill** [comment]
I have submit the pdf of my answering. Thank you very much!
________________________________
发件人: Matthew Lundy <notifications@github.com>
发送时间: 2020年9月22日 0:51
收件人: JIAO-Hao-mcgill/JIAO-Hao-HW <JIAO-Hao-HW@noreply.github.com>
抄送: Hao Jiao <hao.jiao@mail.mcgill.ca>; Comment <comment@noreply.github.com>
主题: Re: [JIAO-Hao-mcgill/JIAO-Hao-HW] Some general issues with assignment 1 (#1)


I have only been going through the grades on q3 at the moment but all questions should have written answers or points will be lost in most cases.

―
